FlashCards.ZubePierreBasali/Database/DataTools.cs

```diff
@@ -0,0 +1,240 @@
+﻿using FlashCards.FlashCardsManager.Models;
+using Spectre.Console;
+using System.Configuration;
+using Dapper;
+using System.Data;
+using System.Data.SqlClient;
+
+using FlashCards.StudySessions;
+
+namespace FlashCards.Database
+{
+    internal class DataTools
+    {
+        internal string ConnectionString { get; set;}
+
+        internal void Initialization()
+        {
+            try
+            {
+                string sqlCommandText;
+                Configuration config = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+                try
+                {
+                    if (config.AppSettings.Settings["connectionString"].Value == "")
+                    {
+                        string connectionString = @$"Data Source=localhost;Integrated Security=SSPI;Initial Catalog=;TrustServerCertificate=True;";
+                        config.AppSettings.Settings["connectionString"].Value = connectionString;
+                        config.Save();
+                        using (SqlConnection connection = new(connectionString))
+                        {
+                            if (connection.State == ConnectionState.Closed) connection.Open();
+                            sqlCommandText = $@"CREATE DATABASE FlashCardsDB";
+                        sqlCommandText = "nejwrnw";
+                            SqlCommand command = connection.CreateCommand();
+                            command.CommandText = sqlCommandText;
+                            command.ExecuteNonQuery();
+                            connection.Close();
+                        }
+                    }
+                }
+                catch(Exception ex) { throw ex; }
+
+                try
+                {
+                    ConnectionString = $"Data Source=localhost;Initial Catalog=FlashCardsDB;Integrated Security=True;TrustServerCertificate=True;";
```

Comment:
🟠 **Hardcoded Connection String**

💡 What is the point of the App.Config if you are just going to ignore it? I don't want my database called `FlashCardsDB`... Now I have to hunt down and make a code change to fix. Instead of it being driven through the config.